### PR TITLE
copr: vectorize is_ipv4_mapped

### PR DESCRIPTION
**Pull request by TommyCpp**
Signed-off-by: zhongyang.wu <zhongyang.wu@outlook.com>

PCP #5751

###  What have you changed?
vectorize `is_ipv4_mapped`

###  What is the type of the changes?

Pick one of the following and delete the others:

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?
Please select the tests that you ran to verify your changes:
- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No.

###  Does this PR affect `tidb-ansible`?
No.

### CODE REVIEW

**Comment by TommyCpp** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/82

File: components/tidb_query/src/rpn_expr/impl_miscellaneous.rs [hunk 1]
```diff
@@ -25,6 +28,23 @@ pub fn is_ipv4_compat(addr: &Option<Bytes>) -> Result<Option<i64>> {
     }
 }
 
+#[rpn_fn]
+#[inline]
+pub fn is_ipv4_mapped(addr: &Option<Bytes>) -> Result<Option<i64>> {
+    match addr {
```
Comment (rleungx) [inline components/tidb_query/src/rpn_expr/impl_miscellaneous.rs]: How about `Ok(match addr {...})`

Comment (TommyCpp) [reply components/tidb_query/src/rpn_expr/impl_miscellaneous.rs]: Sounds good, I will change it

**Comment by breeswish** [comment]
@hicqu PTAL, thanks!

**Comment by breeswish** [comment]
Please fix conflicts. Thanks!

**Comment by TommyCpp** [comment]
Fixed the conflicts

**Comment by TommyCpp** [comment]
Thanks!

**Pull request merged**